### ISSUE TITLE
External Equity Links should be added in the Free Columns

### ISSUE THREAD

**Issue by SuperFexy** [seomoz/SEOmozAPISamples#18]
I'm using the Python library and when I look at the `lsapi.py` file I see that the [externalLinks](https://github.com/seomoz/SEOmozAPISamples/blob/master/python/lsapi.py#L49) column (with the BitFlag of 32) says it's not available to users with a free account.  However, when I reference your API documentation it shows that it is available with a free account.  [See here](https://moz.com/help/guides/moz-api/mozscape/api-reference/url-metrics#cols)

Can we add this to the `lsapi.UMCols.freeCols` attribute?

**Comment by SuperFexy** [comment]
I see that `externalLinks` is [included in the `lsapi.UMCols.freeCols` tuple](https://github.com/seomoz/SEOmozAPISamples/blob/master/python/lsapi.py#L213).  But I don't see an attribute in the response that corresponds to it:

```python
metrics = l.urlMetrics(['www.moz.com', 'seriouseats.com'], lsapi.UMCols.freeCols)
print metrics

[{'fmrp': 5.837410743247288,
  'fmrr': 1.0137685781330276e-07,
  'pda': 91.46535716211868,
  'uid': 4859,
  'ulc': 1455392795,
  'umrp': 6.089938967209491,
  'umrr': 3.1160789797773323e-09,
  'upa': 73.25686321997019,
  'us': 301,
  'ut': u'',
  'uu': 'www.moz.com/'},
 {'fmrp': 5.92159020570692,
  'fmrr': 1.257740684586365e-07,
  'pda': 84.46559166894879,
  'uid': 46617,
  'ulc': 1453560451,
  'umrp': 6.320415213589495,
  'umrr': 7.250263171269388e-09,
  'upa': 71.82088128899844,
  'us': 200,
  'ut': 'Serious Eats: The Destination for Delicious | Serious Eats',
  'uu': 'seriouseats.com/'}]
```

I see the attribute `uid`, but according to your documentation that should be the [number of `Links`](The number of links (equity or nonequity or not, internal or external) to the URL) (The number of links (equity or nonequity or not, internal or external) to the URL).  The `External Equity Links` should have the attribute `ueid` (The number of external equity links to the URL).

**Comment by dlecocq** [comment]
That's interesting. I'll try to reproduce it. FWIW, this code is also available under https://github.com/seomoz/linkscape-py

**Comment by dlecocq** [comment]
Figured out the issue. I'll get a pull request going, but in the mean time, the workaround is to say:

```python
metrics = l.urlMetrics(['www.moz.com', 'seriouseats.com'], lsapi.UMCols.freeCols | 32)
```

**Comment by b4hand** [comment]
Fixed by #19.
